packages/interaction-kit/src/commands/context-menu.ts

```diff
@@ -12,10 +12,10 @@ type ContextMenuArgs<T extends ContextMenuApplicationCommandType> = {
 	name: string;
 	type: T;
 	defaultPermission?: boolean;
-	handler: (
+	onInteraction: (
 		interaction: ContextMenuInteraction<T>,
 		application: Application
-	) => unknown;
+	) => void;
```

Comment:
Might want to make this `Awaitable<void>` (so void | Promise<void>)

Reply:
done, everywhere